packages/ckeditor5-html-support/tests/htmlcomment.js

```diff
@@ -222,6 +223,263 @@ describe( 'HtmlComment', () => {
 		} );
 	} );
 
+	describe( 'createHtmlComment()', () => {
+		it( 'should create an HTML comment between elements', () => {
+			editor.setData( '<p>Foo</p><p>Bar</p><p>Baz</p>' );
+
+			editor.model.change( writer => {
+				const root = editor.model.document.getRoot();
+				const position = writer.createPositionAt( root, 1 );
+
+				htmlCommentPlugin.createHtmlComment( position, 'first' );
+			} );
```

Comment:
I think that changing the model using a callback to add a comment is not needed anymore, since we have an API.

```suggestion
			htmlCommentPlugin.createHtmlComment( model.createPositionAt( root, 1 ), 'first' );
```

Similar change could be made in other tests.